monai/transforms/post/array.py

```
@@ -54,6 +61,7 @@
     "SobelGradients",
     "VoteEnsemble",
     "Invert",
+    "GenerateHeatmap",
     "DistanceTransformEDT",
 ]
```

Ruff (0.13.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)